src/swc/aeon/io/video.py

```diff
@@ -44,11 +45,11 @@ def frames(data: pd.DataFrame) -> Iterable[MatLike]:
 def export(frames: Iterable[MatLike], filename: str, fps: float, fourcc: int | None = None) -> None:
     """Exports the specified frame sequence to a new video file.
 
-    :param iterable frames: An object to iterate over the raw video frame data.
-    :param str filename: The path to the exported video file.
-    :param float fps: The frame rate of the exported video.
-    :param optional fourcc:
-        Specifies the four character code of the codec used to compress the frames.
+    Args:
+        frames: An object to iterate over the raw video frame data.
+        filename: The path to the exported video file.
+        fps: The frame rate of the exported video.
+        fourcc: Specifies the four character code of the codec used to compress the frames.
```

Comment:
Should we consider changing this to accept a str instead of int if we anyway use `cv2.VideoWriter.fourcc()` ?